examples/fluids/index.md

```diff
@@ -263,7 +263,26 @@ For the Navier-Stokes and Euler equations in primitive variables, {cite}`whiting
 2. momentum stabilization $\tau_m$
 3. energy stabilization $\tau_E$
 
-while we follow {cite}`hughesetal2010` in defining a $3\times 3$ diagonal stabilization according to spatial criterion 2 (equation 27), since our equations are in conservative form.
+However, since our equations are in conservative form, we follow {cite}`hughesetal2010` in defining a $3\times 3$ diagonal stabilization according to spatial criterion 2 (equation 27) as follows.
+
+$$
+\bm \tau = \frac{ 2 c_{\tau} \xi(\mathrm{Pe})}{\rho(\diff \bm F) \lVert \nabla_{\bm x} \bm X \rVert}
```

Comment:
@jedbrown , is this the correct vector notation for tau?

Reply:
I pushed an update here. `\diff\bm F` is an "increment" or "variation" in `\bm F`, which has the same shape as `\bm F` (a 5x3 matrix). We want the 5x5 matrices for each directional split. I pushed a revision of this section.

Reply:
Thanks for the fix! For some reason, I thought we wanted to stick with vector notation.

Reply:
We can't talk about the eigenvalues or spectral radius of a vector -- that needs to be a matrix. But we don't have to form the matrix because we know its eigenvalues analytically. Our code will work in incremental form (only dealing with state vectors and state fluxes), but we'll use this analytic property of the flux Jacobian.